packages/core/src/components/Counter/__stories__/Counter.stories.tsx

```diff
@@ -52,115 +32,96 @@ export const Overview = {
 
 export const Sizes = {
   render: () => (
-    <>
-      <div className="storybook-counter_column">
+    <Flex gap={160}>
+      <Flex direction="column" gap="large" style={{ width: "100px" }} align="start">
         <Counter count={5} size="xs" />
         XS
```

Comment:
Change the strings, so they would use `<Text>` and also change between themes in Storybook